invirtualenv/deploy.py

```diff
@@ -223,7 +231,9 @@ def build_deploy_virtualenv(arguments=None, configuration=None, update_existing=
             )
 
     if version:
-        arguments.name += '_' + version
+        # NOTE(saga) RPMs look like <name>-<version>.rpm. Just follow the same
+        # pattern everywhere.
+        arguments.name += '-' + version
```

Comment:
Changing this would break existing deploy.conf files.

Reply:
Done! Fixed